docs/how-to-monitor-minio-with-prometheus.md

```diff
@@ -33,35 +33,42 @@ The Prometheus monitoring server
 
 Refer [Prometheus documentation](https://prometheus.io/docs/introduction/first_steps/) for more details.
 
-### 2. Prometheus configuration
+### 2. Generate Prometheus Config with mc
 
-Prometheus configuration is written in YAML. Add MinIO server details to the config file under `scrape_configs` section
+As Prometheus endpoint in MinIO is authorized. For a secured access using a bearer token approach, override the default prometheus config with the one generated using mc.
+To generate a prometheus config for an alias, use [mc](https://docs.min.io/docs/minio-client-quickstart-guide) as follows `mc admin prometheus generate <alias>`.
+
+The command will generate the `scrape_configs` section of the prometheus.yml as follows
 
 ```yaml
 scrape_configs:
-  - job_name: minio
-    metrics_path: /minio/prometheus/metrics
-    static_configs:
-      - targets: ['localhost:9000']
+- job_name: minio-job
+  bearer_token: <secret>
+  metrics_path: /minio/prometheus/metrics
+  scheme: http
+  static_configs:
+  - targets: ['localhost:9000']
 ```
 
-Note that `localhost:9000` is MinIO server instance address, you need to change it to appropriate value in your configuration file.
 
-### 3. Start Prometheus
+### 3. Update `scrape_configs` section in prometheus.yml
+
+To authorize every scrape request, copy and paste the generated `scrape_configs` section in the prometheus.yml and restart the prometheus service.
 
-Start Prometheus by running
+### 4. Start Prometheus
+
+Start (or) Restart Prometheus service by running
 
 ```sh
 ./prometheus --config.file=prometheus.yml
 ```
-
 Here `prometheus.yml` is the name of configuration file. You can now see MinIO metrics in Prometheus dashboard. By default Prometheus dashboard is accessible at `http://localhost:9090`.
 
 ## List of MinIO metric exposed
```

Comment:
I will send a seperate PR for the latest list of metrics. After the refactor PR gets merged.